fastlanez/src/bitpack.rs

```diff
@@ -16,6 +16,11 @@ where
     Pred<{ W > 0 }>: Satisfied,
     Pred<{ W < 8 * size_of::<Self>() }>: Satisfied,
 {
+    // 1024 elements compress to W bits each -> (1024 * W / 8) bytes
+    const BYTES_PER_TRANCHE: usize = 128 * W;
```

Comment:
We don't really use the word tranche anywhere, maybe just stay consistent and inline 128 * W.

Same for below.